index.js

```diff
@@ -45,7 +45,8 @@ function loadPrismLang(lang) {
 	let langObject = Prism.languages[lang];
 	if (langObject === undefined) {
 		try {
```

Comment:
I think this `try … catch` should be removed now. It was used to catch errors thrown by `require` if a language did not exist.

But if the new `require` failed, this would indicate a serious error which we should not catch.